engine/src/desktop-ans.cpp

```diff
@@ -60,7 +60,7 @@ int MCA_folder(MCStringRef p_title, MCStringRef p_prompt, MCStringRef p_initial,
 	MCPlatformWindowRef t_owner;
     t_owner = compute_sheet_owner(p_options);
 	
-	MCPlatformBeginFolderOrFileDialog(kMCPlatformFileDialogKindFolder, t_owner, p_title, p_prompt, nil, 1, p_initial, false);
+	MCPlatformBeginFolderOrFileDialog(kMCPlatformFileDialogKindFolder, t_owner, p_title, p_prompt, nil, 1, p_initial);
```

Comment:
If the p_types parameter is nullptr, shouldn't the type_count be 1? (I suspect there's a nullptr guard in the function, but should probably make it correct in the caller here).